src/libtriton/CMakeLists.txt

```diff
@@ -7,7 +7,7 @@ endif()
 
 # Default flags
 # To fix old python version bug http://bugs.python.org/issue21958
-if(${CMAKE_SYSTEM_NAME} MATCHES "Windows")
+if(${CMAKE_SYSTEM_NAME} MATCHES "Windows" AND NOT "${CMAKE_CXX_COMPILER_ID}" MATCHES "GNU")
   set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -DHAVE_ROUND /MT")
 endif()
 
```

Comment:
Thanks for the report, I didn't really understand this warning under windows, I will do the change.

About the CRT_SECURE, I would prefer keep the warning and fix them than silent them all as the concern may be real. You report here the fopen call, did you see any other? appveryor didn't report them AFAIK